### PR TITLE
Claw changes 

### PR DESCRIPTION
**Pull request by mastermind-mayhem**
Creating Claw changes for use of a relative encoder

### CODE REVIEW

**Review by awesomeatpiano — APPROVED** [review]
cool beans :)

File: src/main/java/frc/robot/subsystems/ClawSubsystem.java [hunk 1]
```diff
@@ -43,10 +62,18 @@ public void clamp() {
   }
 
   public void release() {
-
     motleft.setInverted(true);
     motright.setInverted(false);
-    clawGroup.set(-0.4);
+    if (getrightenc() >= 0) {
```
Comment (mathyourlife) [inline src/main/java/frc/robot/subsystems/ClawSubsystem.java:67]: TODO: We'll need to make sure that encoder values increase as the claws move inward.  (may need to invert the values)

File: src/main/java/frc/robot/subsystems/ClawSubsystem.java [hunk 2]
```diff
@@ -15,12 +16,30 @@ public class ClawSubsystem extends SubsystemBase {
...
-  private final CANSparkMax motright = new CANSparkMax(13, MotorType.kBrushed);
-  private final DutyCycleEncoder leftenc = new DutyCycleEncoder(0);
-  private final DutyCycleEncoder rightenc = new DutyCycleEncoder(1);
+  private final CANSparkMax motright = new CANSparkMax(13, MotorType.kBrushless);
+  private final RelativeEncoder leftenc = motleft.getEncoder();
+  private final RelativeEncoder rightenc = motright.getEncoder();
 
   MotorControllerGroup clawGroup = new MotorControllerGroup(motleft, motright);
 
+  public void zeroclaw() {
```
Comment (mathyourlife) [inline src/main/java/frc/robot/subsystems/ClawSubsystem.java:25]: Leave a comment that the claw subsystem expects claws to start in the full open position when zeroed.

**Review by mathyourlife — APPROVED** [review]
LGTM

**Review by mathyourlife — APPROVED** [review]

**Pull request merged**